server/src/main/java/org/loomdev/loom/command/CommandManagerImpl.java

```diff
@@ -64,10 +64,7 @@ private CommandSource getSource(@NotNull CommandSourceStack stack) {
 
     @Override
     public void register(@NotNull Object plugin, @NotNull Command command) {
-        var container = server.getPluginManager().fromInstance(plugin);
-        if (container != null) {
-            register(container.getMetadata(), command);
-        }
+        server.getPluginManager().fromInstance(plugin).ifPresent(container -> register(container.getMetadata(), command));
```

Comment:
Wrap for readability (below as well)

Reply:
A general rule I follow for clean function programming is to put each function call on its own line. What do you think about that?